docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - ./init.sql:/docker-entrypoint-initdb.d/init.sql
     ports:
       - "5432:5432"
+    env_file: .env
 
```

Comment:
Docker was not loading env vars for Postgres. We still do not know why it worked without, but this fixes it